### PR TITLE
setup calculator wire frames

### PR DESCRIPTION
**Pull request by Victor-chiemerie**
- Delete all the boilerplate from Create React App.
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches the given Calculator design:
- Implement only JSX and CSS design
- Develop just the UI of the calculator.
   - Use functional components
- Create at least 1 child component
- Test it in the browser; it renders without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <div className="function">-</div>
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
+    <div className="function">+</div>
+    <div className="zero">0</div>
+    <div>.</div>
+    <div className="function">=</div>
+  </section>
+);
```
Comment (roshan-bajgain) [inline src/components/Calculator.js]: - [ ] According to the project requirement `Calculator.js` should have at least 1 child component. Please implement it.

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+    //     <a
+    //       className="App-link"
+    //       href="https://reactjs.org"
+    //       target="_blank"
+    //       rel="noopener noreferrer"
+    //     >
+    //       Learn React
+    //     </a>
+    //   </header>
+    // </div>
```
Comment (roshan-bajgain) [inline src/App.js]: - [ ] [Optional] Please remove the commented code as it make the code dirty.

File: src/components/Calculator.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  background-color: rgb(78, 82, 93);
+}
+
+.zero {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.function {
+  background-color: rgb(225, 115, 56);
```
Comment (roshan-bajgain) [inline src/components/Calculator.css]: - [ ] Your component should match the following given design: please make sure that it matches the design.
- [ ] Please use the same color on the screen, keyword.
### Your design
his

### Original design
original

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Victor-chiemerie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] Please remove All the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) in public folder.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <div>1</div>
+      <div>2</div>
+      <div>3</div>
+      <div className="function">+</div>
+      <div className="zero">0</div>
+      <div>.</div>
+      <div className="function">=</div>
+    </div>
+  </section>
+);
```
Comment (Owusu-Desmond) [inline src/components/Calculator.js]: - [ ] As requested by the previous code reviewer, from the project requirement `Calculator.js` should have at least 1 child component. So please make sure that you create a child component for this Calculator component. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Victor-chiemerie  :wave: 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Victor-chiemerie , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**